sky/tools/sky_snapshot/vm.cc

```diff
@@ -21,7 +21,7 @@ void InitDartVM() {
   CHECK(Dart_SetVMFlags(arraysize(kDartArgs), kDartArgs));
   CHECK(Dart_Initialize(blink::kDartVmIsolateSnapshotBuffer, nullptr, nullptr,
                         nullptr, nullptr, nullptr, nullptr, nullptr, nullptr,
-                        nullptr, nullptr) == nullptr);
+                        nullptr, nullptr, nullptr) == nullptr);
```

Comment:
This function is getting out of control.  Can we switch to a struct or something?